### PR TITLE
[tsql] Support for "create or replace trigger"-syntax observed working on Sybase ASE 16.

### PR DESCRIPTION
**Pull request by olebhansen**
This is a resubmission of one of the the changes from cancelled PR https://github.com/antlr/grammars-v4/pull/3196, adding more support for Sybase. This should not require the rework suggested for the other change included in that PR.

### CODE REVIEW

File: sql/tsql/TSqlParser.g4 [hunk 1]
```diff
@@ -2192,7 +2192,7 @@ create_or_alter_trigger
     ;
 
 create_or_alter_dml_trigger
-    : ((CREATE (OR ALTER)?) | ALTER) TRIGGER simple_name
+    : ((CREATE (OR (ALTER | REPLACE))?) | ALTER) TRIGGER simple_name
```
Comment (KvanTTT) [inline sql/tsql/TSqlParser.g4:2195]: Please remove excess parentheses:

```suggestion
    : (CREATE (OR (ALTER | REPLACE))? | ALTER) TRIGGER simple_name
```

File: sql/tsql/TSqlParser.g4 [hunk 2]
```diff
@@ -2212,7 +2212,7 @@ dml_trigger_operation
     ;
 
 create_or_alter_ddl_trigger
-    : ((CREATE (OR ALTER)?) | ALTER) TRIGGER simple_name
+    : ((CREATE (OR (ALTER | REPLACE))?) | ALTER) TRIGGER simple_name
```
Comment (KvanTTT) [inline sql/tsql/TSqlParser.g4:2215]:
```suggestion
    : (CREATE (OR (ALTER | REPLACE))? | ALTER) TRIGGER simple_name
```

**Review by KvanTTT — CHANGES_REQUESTED** [review]

**Comment by olebhansen** [comment]
@KvanTTT ; I've addressed your comments in the 62e3fb6 commit, also fixing the indentation in the test that I caught after doing the initial push/PR.

**Review by KvanTTT — APPROVED** [review]

**Comment by kaby76** [comment]
Note, the build failed because the grouped parsing took over 5 minutes. I am nearing completion of a fix for to use individual parsing, and having multiple test scenarios per grammar in a day or two. So ignore these types of errors for now. https://github.com/antlr/grammars-v4/pull/3195

**Pull request merged**